man/man7/fi_rxm.7

```diff
@@ -151,7 +151,7 @@ would be read per progress (RxM CQ read).
 .B \f[I]FI_OFI_RXM_SAR_LIMIT\f[]
 Set this environment variable to control the RxM SAR (Segmentation And
 Reassembly) protocol.
-Messages of size greater than this (default: 256 Kb) would be
```

Comment:
Only update the .md file.  The man page will be auto-generated.

Reply:
@shefty Done.